data/datacatalogue/molgenis.csv

```diff
@@ -231,7 +231,8 @@ Formats,OntologyTerms,,,,,,,,,,
 InclusionCriteria,OntologyTerms,,,,,,,,,,
 Keywords,OntologyTerms,,,,,,,,,,Used to generate the tree on the left; we might want to make multiple trees?
 Data,OntologyTerms,,,,,,,,,,Used to generate the tree on the left; we might want to make multiple trees?
-AgeCategories,OntologyTerms,,,,,,,,,,e.g. '8 week'
+AgeCategories,OntologyTerms,,,,,,,,,,Participant and population age categories
+AgeRanges,OntologyTerms,,,,,,,,,,
```

Comment:
What is meaning of ageRanges?

Reply:
to splt age categories (age groups), such as adolescent, adult etc and the possibility to enter custom ranges, which may be appropriate for collection events